### PR TITLE
Add harshraj8843

### PR DESCRIPTION
**Pull request by harshraj8843**
Fixes #

## Proposed Changes

  - update footer (get latest release version through github api) 
  - add harshraj8843
  - update `.gitignore` for `yarn.lock`
  - update release version in `public/app.json`

## Screenshots

Screenshot from 2021-09-19 21-20-04
Screenshot from 2021-09-19 21-20-18

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/250"></a>

### CODE REVIEW

File: public/app.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "version": "0.0.0"
+  "version": "0.16.0"
```
Comment (eddiejaoude) [inline public/app.json:2]: this is automatically updated by a GitHub Action - changing this will break the GitHub Action because it looks for `0.0.0`

File: src/Components/Footer.js [hunk 2]
```diff
@@ -3,9 +3,10 @@ import React, { useState, useEffect } from 'react'
 function Footer() {
   const [version, setVersion] = useState('')
   useEffect(() => {
-    fetch('/app.json')
+    // latest release
+    fetch('https://api.github.com/repos/EddieHubCommunity/LinkFree/releases')
       .then((response) => response.json())
-      .then((data) => setVersion(data.version))
+      .then((data) => setVersion(data[0].tag_name))
```
Comment (eddiejaoude) [inline src/Components/Footer.js:9]: oh this is an interesting idea 👍

File: public/data/harshraj8843.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/harshraj8843",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (eddiejaoude) [inline public/data/harshraj8843.json:18]: I would put this in another PR so it can be merged and not blocked by other changes that might need more discussion

File: public/app.json (hunk 1 above)
Comment (harshraj8843) [reply public/app.json:2]: Should i push another commit to revert it ?

Comment (harshraj8843) [reply public/app.json]: Done !!!

File: public/data/harshraj8843.json (hunk 3 above)
Comment (harshraj8843) [reply public/data/harshraj8843.json:18]: Thanks

File: .gitignore [hunk 4]
```diff
@@ -24,3 +24,6 @@ yarn-error.log*
 
 # app
 public/list.json
+
+# yarn
+yarn.lock
```
Comment (Panquesito7) [inline .gitignore:29]: Missing an endline.
```suggestion
yarn.lock

```

File: public/data/harshraj8843.json (hunk 3 above)
Comment (Panquesito7) [inline public/data/harshraj8843.json:18]: Missing an endline.
```suggestion

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/harshraj8843.json (hunk 3 above)
Comment (Panquesito7) [reply public/data/harshraj8843.json:18]: I agree with Eddie. 👍

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
I would recommend making separate pull requests for your profile addition and for the new feature/idea you want to add. Thank you. 🙂
